nextroute/schema/output.go

```diff
@@ -6,60 +6,96 @@ import (
 	"github.com/nextmv-io/sdk/nextroute/check"
 )
 
-// SolutionOutput represents a solutions as JSON.
+// SolutionOutput represents a solution as JSON.
 type SolutionOutput struct {
-	Unplanned []StopOutput    `json:"unplanned"`
-	Vehicles  []VehicleOutput `json:"vehicles"`
+	// Unplanned is the list of stops that were not planned in the solution.
+	Unplanned []StopOutput `json:"unplanned"`
+	// Vehicles is the list of vehicles that were used in the solution.
+	Vehicles []VehicleOutput `json:"vehicles"`
+	// Objective is the objective of the solution.
```

Comment:
Isn't this statistics / information about the solution's objective value? The "objective of the solution" makes it sound like it defines the models objective. :thinking: